<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>objectOverview</class>
 <widget class="QWidget" name="objectOverview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип сделки</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_deal">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Выберите</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_extended">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_address">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>154</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Район</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_district">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип недвижимости</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>225</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена, USD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_price">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Риелтор</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_rieltor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Выберите</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>255</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Площадь, м2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_square">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_floor_amount">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во комнат</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_year">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Год строительства</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_client">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Выберите</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Клиент</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deal">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Заключить сделку</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_type">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_deal">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сделка</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
